### PR TITLE
Added 2MHz and 16MHz internal PLL clocks.

### PR DESCRIPTION
**Pull request by xcvista**
I was implementing bit-banged 1-Wire so a FAST clock is a must-have. So I added the 16MHz one. The 2MHz one is more or less a "buy one get one free" as it is controlled by the CKDIV8 fuse.

### CODE REVIEW

**Comment by damellis** [comment]
Sounds good. Can you summarize the implications of using a PLL based clock?

**Comment by xcvista** [comment]
@damellis It takes slightly longer for the MCU to start as it requires the PLL to properly lock. I think there are also a little bit of code required in `setup()` regarding PLL.

**Comment by damellis** [comment]
The smaller start up time shouldn't be a big deal, but if this requires additional code in the user sketch, that's definitely something we'd want to explain. Do you have the code?

**Comment by xcvista** [comment]
Not yet. I need to do some investigation on whether it need startup code.

**Comment by sonyhome** [comment]
Please push this change. It is useful to have a 16MHz attiny85.
This change supersedes the previous pull request on your code branch for 16MHz that also has bugs in the definition.
I am not sure what code is required in setup() per previous comments?

I have all my attinys running 16MHz.
I would recommend setting brownout detection and self programming.
I say this change can be pulled as-is otherwise. :)

**Comment by damellis** [comment]
sonyhome: does the 16 MHz internal PLL clock require any additional code in setup() or other changes to the user's code?

**Comment by xcvista** [comment]
@damellis So far I did not find the need of any startup code.

**Comment by damellis** [comment]
Great, thanks! Is there anything else that people need to be aware of when using the PLL clock? Or can we just call it 16 MHz internal and not bother people with the distinction between PLL and non-PLL?

**Comment by xcvista** [comment]
@damellis I would keep calling it 16MHz PLL as it does have a slower start than non-PLL frequencies. Also with the PLL running already it is simpler to put PWM module into the high frequency PWM mode (64MHz)

**Comment by damellis** [comment]
One problem is that the PLL doesn't exist on the ATtiny44/84 (I believe). So we'll need to modify this to only apply for the ATtiny45/85. That might require us to make two separate top-level entries in the boards menu, one for the ATtiny45/85 and one for the ATtiny44/84.

**Comment by sonyhome** [comment]
Sorry to be so slow to respond:

There is no code needed to use the PLL on AtTiny85. Just set the fuses and
go.

AtTiny84 does not seem to have a built-in PLL to clock on, so yes you have
to special case the PLL mode to the 85 only.
